### ISSUE TITLE
Website does not function when Ad Block is enabled

### ISSUE THREAD

**Issue by Septem151** [toptal/gitignore.io#576]
## Issue

The website does nothing when adblocker is on. Clicking the "Create" button does nothing.

### Development Environment:

- [ ] Machine (Local, Container, Virtual):
- [ ] Operating System (Name/Version):
- [x] Web Browser: Brave Browser Version 1.37.111 Chromium: 100.0.4896.79 (Official Build) (64-bit)
- [ ] Vapor Version:
- [ ] Swift Version:

### Expected Behavior:

Clicking "Create" on the website should redirect me to the generated gitignore file.

### Actual Behavior:

Clicking "Create" does nothing. No errors reported in console, no network activity.

### Reproduction Steps:

Open a web browser to either https://www.toptal.com/developers/gitignore/ or https://gitignore.io and have any Ad Blocker enabled (in my case, ublock origin and the built-in Brave browser Ad Block). Try to click "Create" with a specified language(s) to generate, watch as nothing happens. Reproducible with Brave's built-in Ad Block disabled and using just ublock origin.

**Comment by kishaningithub** [comment]
Same is also reproducible in firefox private windows and normal windows with ublock enabled.

The only way i am using this site is via incognito window in google chrome which does not have any addons enabled.

**Comment by jhult** [comment]
I can't get the "Create" button to work in any browser (Safari, Firefox, Chromium) even without any adblocking extensions.

**Comment by Shaquu** [comment]
Same on Chrome with AdBlock. Thanks for ruining the tool :)

**Comment by dunefox** [comment]
It's not working for me either, regardless of extensions and the browser itself.

**Comment by ravgeetdhillon** [comment]
Facing the same issue. But works in Incognito mode.

**Comment by rumdood** [comment]
Same issue. Doesn't work in any browser - not even in Chrome with incognito mode. Tried disabling PiHole to see if that corrected the issue but still doesn't work.

Windows 10: 19044.1645
Firefox 99.0.1
Chrome 100.0.4896.127

**Comment by joeyshi12** [comment]
Going into incognito mode doesn't work for me. It works when I disable all my add-ons on firefox.

Another way to get your gitignore file is to call their endpoint directly. For example,
`wget https://www.toptal.com/developers/gitignore/api/python,vim .gitignore` for python+vim

You can list the available options with
`curl https://www.toptal.com/developers/gitignore/api/list`

**Comment by k30v1n** [comment]
Same... is this a bug or a feature? 👁️

**Comment by rungxanh1995** [comment]
I just checked, and it's working with Chrome and Firefox (no need to disable any add-ons), but not Safari 😂

**Comment by rossheat** [comment]
Got the create button to work in Chrome Incognito mode. Doesn't work in regular mode. Possibly related to uBlock origin.

**Comment by RobertNickRevnic** [comment]
Tested in Brave Browser [Version 1.38.111 Chromium: 101.0.4951.54 (Official Build) (64-bit)]
It works in a private window with the Brave shields down (it has an internal adblocker), but in a regular window, it does not work, even with all the adblockers disabled.

Tested in Chrome [Version 101.0.4951.54 (Official Build) (64-bit)]
It works in normal and private windows, as long as the adblocker is disabled.

**Comment by wwestlake** [comment]
I'm having this issue in Chrome with U-Block Origin turned on.  With it turned off, the site works fine.  In the console, I see this error:

GET https://static.cloudflareinsights.com/beacon.min.js/v652eace1692a40cfa3763df669d7439c1639079717194 net::ERR_BLOCKED_BY_CLIENT

**Comment by nesevis** [comment]
Currently not working with network-wide ad blocking. Applies to both Safari and Firefox.

`main.js` is successfully loaded.

**Comment by iPaulPro** [comment]
Why'd you have to ruin a good thing, Toptal? Change the name back and get rid of these issues.

**Comment by ttimasdf** [comment]
I wrote this user script to bypass the misuse of Google Analytics. The code is hidden behind event report callback. 👎 

```js
(function() {
    'use strict';
 
    unsafeWindow.gtag = (event, ...args) => {
        var cb = args[1].event_callback;
        if (typeof cb === "function") {
            cb();
        }
        console.log(event, ...args);
    };
})();
```

The userscript could be installed on [GreasyFork - Emulate Google Analytics on gitignore.io](https://greasyfork.org/scripts/446172). This script would fix #577 too.

**Comment by Sn0wfreezeDev** [comment]
Tested with Safari 15.5 and AdGuard enabled
